apps/condo/domains/user/utils/ensureNotExists.js

```diff
@@ -0,0 +1,12 @@
+const { User: UserServerUtils } = require('@condo/domains/user/utils/serverSchema')
+const { STAFF } = require('@condo/domains/user/constants/common')
+
+async function ensureNotExists (context, field, value) {
+    const existed = await UserServerUtils.getAll(context, { [field]: value, type: STAFF })
+    if (existed.length !== 0) {
+        throw new Error(`[unique:${field}:multipleFound] user with this ${field} is already exists`)
+    }
+}
```

Comment:
Do we need this function? Doesnt the same logic already implemented in condo/domains/user/schema/user ?